src/data/Api.js

```diff
@@ -0,0 +1,19 @@
+const list = [];
+
+const hello = async () => {
+  let url = "http://localhost:3001/list/all";
+  const setHeaders = { headers: { "Content-Type": "application/json" } };
+  let myresult = await fetch(url, setHeaders);
+  myresult = await myresult.json();
+  //console.log(myresult);
```

Comment:
Remove unused consoles throughout the app.

Reply:
done.